swingset/src/main/java/com/nqadmin/swingset/SSDBComboBox.java

```diff
@@ -1549,19 +1526,17 @@ public void setSelectedStringValue(final String _value) {
 	 */
 	public void setSelectedMapping(final long _value) {
 
-		// HANDLED BY UNDERLYING INFRASTRUCTURE: TODO consider firing a property change
-		// HANDLED: TODO what if user pass null or if getSelectedValue() is null?
-
+		// TODO: IS THIS COMMENT CORRECT? multiple items with same mapping?
 		// 2020-08-03: Removing conditional as this could be called when consecutive records
```

Comment:
I agree. I don't think that 2020-08-03 comment is right. May have been copy/paste or have mixed up value/items/mappings/options. Would not expect duplicate mappings.